interface/VideoEncoderDefs.h

```diff
@@ -152,6 +152,11 @@ typedef struct VideoResolution {
     uint32_t height;
 }VideoResolution;
 
+typedef struct EncMiscParameterHRD {
```

Comment:
We already have VideoParamsHRD, and do not put it to common, not every one need set this value